solr_conf/conf/managed-schema

```diff
@@ -342,7 +350,7 @@
   <copyField source="tags_ss_lower" dest="tags_t" maxChars="30000" />
   <copyField source="type_s" dest="type_t" maxChars="300" />
 
-  <!-- copy description and transcription to nostem fields for exact matches -->
+  <!-- copy description and transcription to nostem and regex fields for exact matches -->
```

Comment:
did you mean to add another copy field for regex or is this an error in the comment? (oof on the complexity here of all the variations and copies)